cypress/e2e/3-default-owner.cy.ts

```diff
@@ -26,90 +26,92 @@ const lastName = randLastName();
 describe('Default owner', () => {
 	// todo test should redirect to setup if have not skipped
 
-	beforeEach(() => {
+	before(() => {
 		cy.resetAll();
 	});
+	beforeEach(() => {
+		cy.skipSetup();
+	})
 
-	it('should be able to use n8n without user management and setup UM', () => {
-		describe('should skip owner setup', () => {
-			cy.skipSetup();
-			cy.url().should('include', workflowsPage.url);
-		});
+	it('should skip owner setup', () => {
+		cy.visit(workflowsPage.url);
```

Comment:
this is not testing the redirect now.. it should test that user got successfully redirected to workflows page

Reply:
I've added cy.skipSetup() to this spec. The redirection is already tested within this command:
```
Cypress.Commands.add('skipSetup', () => {
	const signupPage = new SignupPage();
	const workflowsPage = new WorkflowsPage();
	const Confirmation = new MessageBox();

	cy.visit(signupPage.url);

	signupPage.getters.form().within(() => {
		cy.url().then((url) => {
			if (url.endsWith(signupPage.url)) {
				signupPage.getters.skip().click();


				Confirmation.getters.header().should('contain.text', 'Skip owner account setup?');
				Confirmation.actions.confirm();

				// we should be redirected to /workflows
				cy.url().should('include', workflowsPage.url); // Make sure user is redected
			} else {
				cy.log('User already signed up');
			}
		});
	});
})

```